src/components/sbb-divider/sbb-divider.tsx

```diff
@@ -0,0 +1,35 @@
+import { Component, h, Prop } from '@stencil/core';
+import { InterfaceSbbDividerAttributes } from './sbb-divider.custom.d';
+
+/**
+ * @slot unnamed - Use this to document a slot.
+ */
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-divider.scss',
+  tag: 'sbb-divider',
+})
+export class SbbDivider {
+  /** Appearance property for displaying the component in dark mode */
+  @Prop() public appearance?: InterfaceSbbDividerAttributes['appearance'] = 'primary';
+
+  /** Orientation property with possible values 'horizontal' | 'vertical'. Defaults to horizontal. */
+  @Prop() public orientation?: InterfaceSbbDividerAttributes['orientation'] = 'horizontal';
+
+  public render(): JSX.Element {
+    const orientationPositionClass = 'sbb-divider--' + this.orientation;
+    const appearanceClass = 'sbb-divider--' + this.appearance;
+
+    return (
+      <div
```

Comment:
There is no need for an internal div. This can be directly applied to the Host: https://stenciljs.com/docs/host-element

Reply:
I refactored div to Host and I made some changes to the .scss file, can you check if the selectors are fine?

Reply:
They are fine :)